### PR TITLE
feat: add ColumnEvalTransformer

### PR DESCRIPTION
**Pull request by chrislemke**

### CODE REVIEW

File: src/sk_transformers/utils.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from sklearn.utils.validation import check_array, check_is_fitted
 
 
-def check_ready_to_transform(
+def  check_ready_to_transform(
```
Comment (premsrii) [inline src/sk_transformers/utils.py:8]: There is an extra space character.

File: src/sk_transformers/generic_transformer.py [hunk 2]
```diff
@@ -46,7 +135,7 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         Returns:
             pandas.DataFrame: Transformed dataframe.
         """
-        check_ready_to_transform(
+        X = check_ready_to_transform(
```
Comment (premsrii) [inline src/sk_transformers/generic_transformer.py:138]: Shouldn't we do this also for `AggregateTransformer`?

**Review by premsrii — CHANGES_REQUESTED** [review]
Check other comments.

File: src/sk_transformers/generic_transformer.py (hunk 2 above)
Comment (chrislemke) [reply src/sk_transformers/generic_transformer.py:138]: I added it to all the missing positions.

File: src/sk_transformers/utils.py (hunk 1 above)
Comment (chrislemke) [reply src/sk_transformers/utils.py]: 🤦‍♂️

**Pull request merged**